ocs_ci/utility/ibmcloud.py

```diff
@@ -1237,3 +1240,112 @@ def find_free_network_subnets(subnet_cidr, network_prefix=27):
         if is_free:
             logger.info(f"Free set of subnets found: {possible_subnets}")
             return possible_subnets
+
+
+class IBMCloudObjectStorage:
+    """
+    IBM Cloud Object Storage (COS) class
+    """
+
+    def __init__(self, api_key, service_instance_id, endpoint_url):
+        """
+        Initialize all necessary parameters
+
+        Args:
+            api_key (str): API key for IBM Cloud Object Storage (COS)
+            service_instance_id (str): Service instance ID for COS
+            endpoint_url (str): COS endpoint URL
+
+        """
+        self.COS_API_KEY_ID = api_key
```

Comment:
Any reason for upper case name of vars?  This is usually used in constants, which is not the case here I think.

Reply:
chnaged to lowercase